src/test/java/com/suse/saltstack/netapi/examples/Calls.java

```diff
@@ -0,0 +1,63 @@
+package com.suse.saltstack.netapi.examples;
+
+import com.suse.saltstack.netapi.AuthModule;
+import com.suse.saltstack.netapi.calls.WheelResult;
+import com.suse.saltstack.netapi.calls.modules.Grains;
+import com.suse.saltstack.netapi.calls.modules.Test;
+import com.suse.saltstack.netapi.calls.wheel.Key;
+import com.suse.saltstack.netapi.client.SaltStackClient;
+import com.suse.saltstack.netapi.datatypes.target.Glob;
+import com.suse.saltstack.netapi.datatypes.target.MinionList;
+import com.suse.saltstack.netapi.datatypes.target.Target;
+import com.suse.saltstack.netapi.exception.SaltStackException;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Example code calling salt modules using the generic interface.
+ */
+public class Calls {
+
+    private static final String SALT_API_URL = "http://localhost:8000";
+    private static final String USER = "saltdev";
+    private static final String PASSWORD = "saltdev";
+
+    public static void main(String[] args) throws SaltStackException {
+        // Init the client
+        SaltStackClient client = new SaltStackClient(URI.create(SALT_API_URL));
+
+        // Ping all minions using a glob matcher
+        Target<String> globTarget = new Glob("*");
+        Map<String, Boolean> results = client.callSync(
+                Test.ping(), globTarget, USER, PASSWORD, AuthModule.AUTO);
+
+        System.out.println("--> Ping results:\n");
+        results.keySet().forEach(
+                key -> System.out.println(key + " -> " + results.get(key)));
+
+        // Get the grains from a list of minions
+        Target<List<String>> minionList = new MinionList("minion1", "minion2");
+        Map<String, Map<String, Object>> grains = client.callSync(
+                Grains.items(true), minionList, USER, PASSWORD, AuthModule.AUTO);
+
+        for (String minion : grains.keySet()) {
```

Comment:
Also here, it could be shortened.
```java
grains.forEach((minion, grain) -> {
    System.out.println("\n--> Listing grains for '" + minion + "':\n");
    grain.forEach((key, val) -> System.out.println(key + ": " + val));
});
```
Just a tip :smile_cat: